core-bundle/src/Command/InstallCommand.php

```diff
@@ -239,12 +246,25 @@ private function addInitializePhp()
      */
     private function symlinkTcpdfConfig()
     {
+        $coreBundle = new \ReflectionClass($this->bundles['ContaoCoreBundle']);
+        $relPath = $this->getRelativePath(\dirname($coreBundle->getFileName()));
```

Comment:
Couldn’t we just call `$this->bundles['ContaoCoreBundle']->getPath()` here? Or is `$this->bundles` a string array of class names?

Reply:
Judging from the tests of the previous PR it seems to be an associative array of class names. Haven't really checked my self, but that's what I remember from `%kernel.bundles%` too.